convert2rhel/checks.py

```diff
@@ -66,6 +73,81 @@ def perform_pre_ponr_checks():
     ensure_compatibility_of_kmods()
 
 
+def check_convert2rhel_latest():
+    """Make sure that we are running the latest version of convert2rhel"""
+
+    repo_content = (
+        "[convert2rhel]\n"
+        "name=Convert2RHEL Repository\n"
+        "baseurl=https://ftp.redhat.com/redhat/convert2rhel/$releasever/os/\n"
+        "gpgcheck=0\n"
```

Comment:
This probably won't matter because we're not actually downloading the package (and so we'll never have the opportunity to check the gpg signatures) but it is good practice to use `gpgcheck=1` everywhere just in case we install the package in some future version of the code.

```suggestion
        "gpgcheck=1\n"
```